src/ec/suite_b/private_key.rs

```diff
@@ -119,6 +119,13 @@ fn private_key_as_scalar_(ops: &PrivateKeyOps, private_key: &ec::PrivateKey)
     Scalar::from_limbs_unchecked(&limbs)
 }
 
+// Are limbs within (0, max_exclusive)?
+fn are_limbs_within_range(limbs: &[Limb], max_exclusive: &[Limb]) -> bool {
```

Comment:
I'll make it take a Scalar for now. I think there's a clean way to share the random generation loop without adding any complication here once specialization gets stabilized, but till then, I don't think it's worth trying too hard for a 3-line method.